spec/features/female_missing_info_feature_spec.rb

```diff
@@ -119,4 +126,39 @@
       wait_for { page.current_path == missing_information_prison_prisoners_path(prison.code) }
     end
   end
+
+  context 'when multiple tabs are opened', :js do
+    let(:complexity) { 'medium' }
+
+    it 'can complete forms in parallel' do
+      visit missing_information_prison_prisoners_path prison.code
+      window = page.open_new_window
+      current = page.current_window
+      within "#edit_#{prisoner_id}" do
+        click_link 'Add missing details'
+      end
+      switch_to_window(window)
```

Comment:
As you wanted a 'do things in current tab' method in capybara, would be really nice to create a helper that does implement this interface i.e. creates a new window/tab, switches to it, yields to a block, and then switches back and destroys the tab. Would make the test really nice and readable too.